ui/StatusQ/src/StatusQ/Components/StatusCommunityCard.qml

```diff
@@ -186,6 +186,9 @@ Rectangle {
             anchors.fill: parent
             radius: d.bannerRadius
             color: root.loaded ? root.communityColor : d.loadingColor2
+
+            // hide when image is loaded to avoid glitches on the edge
+            visible: image.status !== Image.Ready
```

Comment:
I guess this will do for now... you could use an `OpacityMask` to cut the corners

Reply:
There is `OpacityMask` used for that. But even in qt's example mask and source are hidden: https://doc.qt.io/qt-5/qml-qtgraphicaleffects-opacitymask.html#example, otherwise we end up with artifacts: 

mask hidden:
![Screenshot from 2022-09-29 23-18-26](https://user-images.githubusercontent.com/20650004/193145203-05f0f9b4-a965-4ca1-b1ba-62b54460ddab.png)

not hidden:
![Screenshot from 2022-09-29 23-19-52](https://user-images.githubusercontent.com/20650004/193145207-2a05e60b-971f-4530-a7f3-73cce96862b0.png)